CHANGELOG.md

```diff
@@ -34,6 +34,21 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 # Changelog
 
+## [Unreleased]
+
```

Comment:
```suggestion
```

I think we can remove these 2 lines on this branch (keep it on master), so that tags/release branches have nice changelogs.